development/custom-image/install-deps-debian.sh

```diff
@@ -39,6 +39,15 @@ sudo add-apt-repository \
 sudo apt update
 sudo apt install -y docker-ce=${DOCKER_VERSION}
 
+# setup logging drived for docker
+echo "{\"log-driver\": \"gcplogs\",\"log-opts\": {\"gcp-meta-name\": \"kyma-integration-test\"}}" > /tmp/daemon.json
```

Comment:
Just a suggestion, how about 
```
cat << EOF > /etc/docker/daemon.json
{
  json structure
 }
```
It would be a bit more readable :)

Reply:
done :)